.cargo/config_fast_builds.toml

```diff
@@ -148,5 +155,7 @@ rustflags = [
 
 # This enables you to run the CI tool using `cargo ci`.
 # This is not enabled by default, you need to copy this file to `config.toml`.
+# Some systems may experience linker performance issues when running doc tests.
+# See https://github.com/bevyengine/bevy/issues/12207 for details.
```

Comment:
I don't think this is the right place for this comment. From how I read it, it looks like you're commenting on the alias and not `rustdoc` itself. I would either move it to where you specify `rustdocflags`, or remove it entirely.